### PR TITLE
Create basic flask application

### PR DESCRIPTION
**Pull request by ProgrammingMuffin**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:focal
+
+RUN apt update
+RUN apt -y install python3 pip mysql-client
+
+RUN mkdir /www
+RUN mkdir /www/app
```
Comment (glanzz) [inline Dockerfile:7]: mkdir /ww/app is enough right..

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+COPY ./ /www/app/
```
Comment (glanzz) [inline Dockerfile:9]: COPY . /www/app/

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+RUN pip install -r /www/app/requirements.txt
```
Comment (glanzz) [inline Dockerfile:11]: Copy requirements.txt to /www/app and then run pip install requirements.txt directly

File: Dockerfile (hunk 1 above)
Comment (ProgrammingMuffin) [reply Dockerfile:7]: image

File: Dockerfile (hunk 3 above)
Comment (ProgrammingMuffin) [reply Dockerfile:11]: https://github.com/ProgrammingMuffin/TaskManager/blob/a7e5af79d81e05e814fc60f7ae1ba51c0bf3bfc0/Dockerfile#L9-L11
I think this change was made in a different PR.

File: Dockerfile (hunk 2 above)
Comment (ProgrammingMuffin) [reply Dockerfile:9]: Added change :) It'll be added in the next PR.

**Pull request merged**